### DISCUSSION TITLE
FormCalc for Amplitude

### DISCUSSION THREAD

**Discussion by Afsoon-KA** [FeynCalc/feyncalc#282] [General]
"Hi, I want to calculate the amplitude. First, I loaded FeynRules, then I wrote a model.fr file and used WriteFeynArtsOutput[]. After that, I installed FeynArts and FormCalc using ./FeynInstall. Then, I loaded them in my Mathematica notebook and used the following code: CreateTopologies[] -> InsertFields[CreateTopologies[]] -> CreateFeynAmp[] -> CalcFeynAmp[]. Everything was fine until I tried to calculate the squared matrix element. However, M$FACoupling does not work, and I encountered an error in my final result for M^2."

ReplaceAll::\!\(\*RowBox[{"{", "M$FACouplings", "}"}]\) is neither a list of replacement rules nor a valid dispatch table, and so cannot be used for replacing.
 Please tell we How can i check it out?

**Comment by vsht** [comment]
Hi,

this forum is about FeynCalc, not FormCalc.

For your questions you should contact the FormCalc developer

https://wwwth.mpp.mpg.de/members/hahn/

Am 01.09.24 um 13:36 schrieb Afsoon-KA:
